src/main/resources/reproduce/msmarco-v1-passage.yaml

```diff
@@ -26,7 +26,7 @@ conditions:
     display: "SPLADE++ EnsembleDistil (cached queries)"
     display_html: "SPLADE++ EnsembleDistil (cached queries)"
     display_row: ""
-    command: bin/run.sh io.anserini.search.SearchCollection -threads $threads -index msmarco-v1-passage.splade-pp-ed -topics $topics -output $output -impact -pretokenized -hits 1000
+    command: java -cp $fatjar io.anserini.search.SearchCollection -threads $threads -index msmarco-v1-passage.splade-pp-ed -topics $topics -output $output -impact -pretokenized -hits 1000
     topics:
       - topic_key: msmarco-v1-passage.dev.splade-pp-ed
         eval_key: tools/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt
```

Comment:
You can replace `tools/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt` with `msmarco-passage.dev-subset` and it should work.

Reply:
And `java -cp $fatjar trec_eval ...` should also work.